src/tasks/DockerRunTaskDefinitionBase.ts

```diff
@@ -21,7 +21,7 @@ export interface DockerContainerPort {
 export interface DockerContainerVolume {
     localPath: string;
     containerPath: string;
-    permissions?: 'ro' | 'rw';
+    permissions?: 'ro' | 'rw' | 'ro,z' | 'rw,z';
```

Comment:
Waiting to hear from @philliphoff if the `,z` option is allowed on Mac. I suspect it is since it works on Linux containers on Windows, but I wanna check for sure...

Reply:
Yep, it works. 👍